include/lbann/comm.hpp

```diff
@@ -864,24 +923,24 @@ class lbann_comm {
    */
   int threads_per_proc;
   /** Pre-allocated buffers for collectives. */
-  std::unordered_map<size_t, std::vector<uint8_t *>> collective_bufs;
+  std::map<size_t, std::vector<uint8_t *>> collective_bufs;
```

Comment:
`m_al_comms` was changed to a `map`, so I figured I might as well change this one too.